packages/forms-web-app/src/views/full-appeal/submit-appeal/proposed-development-changed.njk

```diff
@@ -16,7 +16,7 @@
     label: {
       text: "Enter the agreed description of development"
     },
-    value: proposedDevelopmentChanged.details,
+    value: descriptionDevelopmentCorrect.details,
     errorMessage: errors['description-development-correct-details'] and {
```

Comment:
this change is from another (merged) PR - rebase to avoid this happening

Reply:
resolved.